backups/omarchy-dotfiles-20260128-235549/.config/omarchy/themes/aether/swayosd.css

```
@@ -0,0 +1,5 @@
+@define-color background-color #111E0B;
```

Biome (2.3.13)

[error] Unexpected unknown at-rule: define-color 

define-color is not a standard CSS at-rule, which may lead to unexpected styling results or failure to interpret the styles as intended.
See MDN web docs for a known list of at-rules.
To fix this issue, consider removing the unknown at-rule.

(lint/suspicious/noUnknownAtRules)